odoo/addons/base/tests/common.py

```diff
@@ -34,17 +31,6 @@ def setUpClass(cls):
             'name': 'Test Partner',
         })
 
-    @classmethod
-    def _use_currency(cls, currency_code):
-        # Enforce constant currency
-        currency = cls._enable_currency(currency_code)
-        if not cls.env.company.currency_id == currency:
-            cls.env.transaction.cache.set(cls.env.company, type(cls.env.company).currency_id, currency.id, dirty=True)
-            # this is equivalent to cls.env.company.currency_id = currency but without triggering buisness code checks.
-            # The value is added in cache, and the cache value is set as dirty so that that
-            # the value will be written to the database on next flush.
-            # this was needed because some journal entries may exist when running tests, especially l10n demo data.
-
```

Comment:
not sure this util is not useful anymore though.
Unless we create a test company in all tests commons, it could be useful if we want to modify the main currency of the current company.

Reply:
As you want then, worst case we bring it back when needed.